packages/ui/src/stories/formkit/Range.stories.ts

```diff
@@ -0,0 +1,28 @@
+import { StoryFn } from '@storybook/vue3'
```

Comment:
That's not vuestic range slider.

<img width="276" alt="image" src="https://github.com/user-attachments/assets/091be6ef-5873-4d0b-b8e2-6afa5f5a1340" />

<img width="932" alt="image" src="https://github.com/user-attachments/assets/5c53b733-e4f8-4ffb-9c25-f59e3aef6233" />